standards/ICRC-2/README.md

```diff
@@ -175,27 +185,34 @@ icrc1_supported_standards : () -> (vec record { name : text; url : text }) query
 
 ## Examples
 
-### Alice deposits tokens to a canister
+### Alice deposits tokens to canister C
 
-1. Alice wants to deposit 100 tokens on an `ICRC-2` ledger to a canister.
-2. Alice calls `icrc2_approve` with `spender` set to the canister's principal and `amount` set to the token amount she wants to deposit (100) plus the transfer fee.
+1. Alice wants to deposit 100 tokens on an `ICRC-2` ledger to canister C.
+2. Alice calls `icrc2_approve` with `spender` set to the canister's default account (`{ owner = C; subaccount = null}`) and `amount` set to the token amount she wants to deposit (100) plus the transfer fee.
 3. Alice can then call some `deposit` method on the canister, which calls `icrc2_transfer_from` with `from` set to Alice's (the caller) account, `to` set to the canister's account, and `amount` set to the token amount she wants to deposit (100).
 4. The canister can now determine from the result of the call whether the transfer was successful.
    If it was successful, the canister can now safely commit the deposit to state and know that the tokens are in its account.
 
-### A canister transfers tokens from Alice's account to Bob's account, on Alice's behalf
+### Canister C transfers tokens from Alice's account to Bob's account, on Alice's behalf
 
-1. A canister wants to transfer 100 tokens on an `ICRC-2` ledger from Alice's account to Bob's account.
-2. Alice previously approved the canister to transfer tokens on her behalf by calling `icrc2_approve` with `spender` set to the canister's principal and `amount` set to the token amount she wants to allow (100) plus the transfer fee.
-3. During some update call, the canister can now call `icrc2_transfer_from` with `from` set to Alice's account, `to` set to Bob's account, and `amount` set to the token amount she wants to transfer (100).
-4. Depending on the result of the call, Bob now has 100 tokens in his account, and Alice has 100 tokens less in her account.
+1. Canister C wants to transfer 100 tokens on an `ICRC-2` ledger from Alice's account to Bob's account.
+2. Alice previously approved canister C to transfer tokens on her behalf by calling `icrc2_approve` with `spender` set to the canister's default account (`{ owner = C; subaccount = null }`) and `amount` set to the token amount she wants to allow (100) plus the transfer fee.
+3. During some update call, the canister can now call `icrc2_transfer_from` with `from` set to Alice's account, `to` set to Bob's account, and `amount` set to the token amount she wants to transfer (100 plus the transfer fee).
```

Comment:
```suggestion
3. During some update call, the canister can now call `icrc2_transfer_from` with `from` set to Alice's account, `to` set to Bob's account, and `amount` set to the token amount she wants to transfer (100).
```
The transfer fee is put into the `fee` field and not added in the `amount` field when Canister c is transfer_from-ing from alice's account to bob's account.